package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@b12/metronome",
-  "version": "1.0.13",
+  "version": "1.0.14",
```

Comment:
I'm a bit confused, in this repo, the last metronome version is `1.0.13`, but in the [crowdsurfing](https://github.com/b12io/crowdsurfing/blob/main/product/package.json#L6) it's `1.1.1`. Should we update the version appropriately? @glebovsky

Reply:
Maybe do a rebase first and then up the version?

Reply:
Not sure if I get it: 
* in the main branch of the Metronome repo, the version of the [package](https://github.com/b12io/metronome/blob/main/package.json#L3) is `0.1.13`
* but in the [crowdsurfing](https://github.com/b12io/crowdsurfing/blob/main/product/package.json#L6) it's `1.1.1`

So, it's confusing, I'm not sure if the next version of the library should `1.1.2` or `0.1.14`?

Reply:
Set the version to `1.1.2`